docs/js/features/odata/odata-v4-client.mdx

```diff
@@ -42,3 +43,91 @@ The generated `$filter` parameter of the URL will be:
 ```sql
 $filter=(/any(a0:((a0/Friends/FirstName eq 'firstName' and a0/Friends/LastName eq 'lastName'))))
 ```
+
+### Expand
+
+#### Expand and Select
+
+In contrast to the OData v2 implementation you have select and expand separately.
+In other words selected properties are not expanded automatically as in v2.
+
+```ts
+People.requestBuilder().getAll()
+    .select(People.LAST_NAME)
+    .expand(People.FRIENDS.select(People.FIRST_NAME,People.ADDRESS_INFO))
+```
+
+In the example above you select the `LAST_NAME` of the root entity and expand the navigation property `FRIENDS`.
+In the expanded entity the selected fields are `FIRST_NAME` and `ADDRESS_INFO`.
+
+The generated URL for this request will be:
+```sql
+/People?$select=LastName&$expand=Friends($select=FirstName,AddressInfo)
+```
+If no select is given all non-navigational properties are included in the response.
+
+#### Sub-queries in Expand
+
+Note that you can crate very complex queries within the expand scope:
+```ts
+People.requestBuilder().getAll()
+    .expand(People.FRIENDS
+      .select(People.FIRST_NAME,People.ADDRESS_INFO)
+      .filter(People.LAST_NAME.equals("Miller"))
+      .orderBy(asc(People.GENDER))
+      .top(1)
+      .skip(1)
+    )
+```
+
+In this example the filter will reduce the friends to be shown.
+The effect of a filter depends whether it is used in or outside of an expand.
+The different cases are explained in [Filters in Expand.](#filterParentChildren)
+
+The URL for the query will be:
+```sql
+/People?$expand=Friends($select=FirstName,AddressInfo;$filter=(LastName eq 'Miller');$skip=1;$top=1)
```

Comment:
the order by is missing

Reply:
good catch. I think we had a bug there. I just copied it from the debug console. https://github.com/SAP/cloud-sdk/pull/504